includes/Core/Permissions/Permissions.php

```diff
@@ -428,9 +429,15 @@ private function check_view_splash_capability( $user_id ) {
 		if ( ! Feature_Flags::enabled( 'dashboardSharing' ) ) {
 			return array( self::AUTHENTICATE );
 		}
-		if ( in_array( 'do_not_allow', $this->check_view_shared_dashboard_capability( $user_id ), true ) ) {
+
+		if ( $this->is_shared_dashboard_splash_dismissed( $user_id ) ) {
+			return array( 'do_not_allow' );
+		}
```

Comment:
Can this really be unconditional like this? What about if someone dismissed this at some point before, no longer has shared access but _can_ authenticate? They would still need to be able to access the splash screen I think.

Reply:
Good point, I'll update that 👍

Reply:
Updated this and the test for an admin as well.